pandas/tests/categorical/test_algos.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+
+import pandas as pd
+import pandas.util.testing as tm
+
+
```

Comment:
you should prob pull the categorical tests out of test_algos.py then

Reply:
For better or worse, `test_algos.py` didn't have any tests for Categorical.

Reply:
hmm, yeah i guess just lots of tests for unique.